Dockerfile

```diff
@@ -14,21 +14,22 @@ RUN set -x \
     python3-venv \
     rsync \
     openssh-client \
-  && apt-get clean
+  && apt-get clean \
```

Comment:
Just from the [Best practices for writing Dockerfiles](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/)

> Official Debian and Ubuntu images automatically run apt-get clean, so explicit invocation is not required.

Thus the `rm -rf /var/lib/apt/lists/*` should be enough.

Reply:
changed it